Utilities/Cleanup-AzureStorageAccounts.ps1

```diff
@@ -0,0 +1,260 @@
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This is a script that performs Azure Storage Cleanup.
+
+.EXAMPLE
+    1. Run script in default mode.
+        In default mode, this script will remove 7+ days old files from -
+        1. boot-diagnostics contaiers. (File extension *.*)
+        2. LISAv2 generated VHDs. (File extension *.vhd. Patterns: "LISAv2 / -osdisk.vhd /AUTOBUILT ")
+        3. Cleanup regions : ALL
+        4. Cleanup storage accounts : ALL
+        5. Cleanup containers : vhds, bootdiagnostics*
+
+    Command: .\Utilities\Cleanup-AzureStorageAccounts.ps1 -SecretFilePath .\XML\AzureSecrets_Test_ONLY.xml
+
+    2. Limit cleanup to specific region and / or specific storage accounts.
+
+    Command: .\Utilities\Cleanup-AzureStorageAccounts.ps1 -SecretFilePath .\XML\AzureSecrets_Test_ONLY.xml `
+        -AzureRegions "westus2,westus" -StorageAccountNames "abc1,abc2"
+
+    3. Remove a specific VHD from all storage accounts / regions.
+
+    Command: .\Utilities\Cleanup-AzureStorageAccounts.ps1 -SecretFilePath .\XML\AzureSecrets_Test_ONLY.xml `
+        -VHDNames "EOSG-AUTOBUILT-ABC.vhd"
+
+    4. Remove VHDs based on string patterns.
+
+    Command: .\Utilities\Cleanup-AzureStorageAccounts.ps1 -SecretFilePath .\XML\AzureSecrets_Test_ONLY.xml `
+        -Patterns "AUTOBUILT,LISAv2"
+
+#>
+
+param(
+    [String] $SecretFilePath,
+
+    # Comma separated storage account names. If empty, All storage accounts in current subscriptions will be selected.
+    [string] $StorageAccountNames,
+
+    # Comma separated skipped Storage account names.
+    [string] $SkippedStorageAccountNames,
+
+    # Comma separated Azure regions. If empty, All regions in current subscriptions will be selected.
+    [string] $AzureRegions,
+
+    # Comma separated values of exact VHD names to clean.
+    [String] $VHDNames,
+
+    # Command separated VHD name patterns to clean.
+    [String] $Patterns = "LISAv2,-osdisk.vhd,AUTOBUILT",
+
+    # Cleanup Age (in Days)
+    [int] $CleanupAgeInDays = 7,
+
+    # Switch to enable logs for skipped files. (Makes execution slower).
+    [switch] $ShowSkippedLogs,
+
+    # Switch to enable dry run mode.
+    [switch] $DryRun
+)
+
+#region Authenticate the powershell session.
+try {
+    if ($SecretFilePath) {
```

Comment:
put '#region Authenticate the powershell session.' before if, and have the same indentation with below '#endregion'

Reply:
Fixed.